tests/func/test_tag.py

```diff
@@ -47,47 +47,50 @@ def test(self):
 
 class TestTagAll(TestDvc):
     def test(self):
-        ret = main(["add", self.FOO, self.BAR])
-        self.assertEqual(ret, 0)
+        with self._caplog.at_level(logging.INFO, logger="dvc"):
```

Comment:
By "original issue" I mean moving `repo.state` alongside `lock`.
`

Reply:
That one was failing because some state-related debug logs started making its way into the caplog, because it wasn't narrowed down to INFO loglevel, so I had to fix that one.